modules/nf-neuro/segmentation/synthseg/main.nf

```diff
@@ -0,0 +1,110 @@
+process SEGMENTATION_SYNTHSEG {
+    tag "$meta.id"
+    label 'process_single'
+    label 'process_high'
+
+    container "freesurfer/freesurfer:7.4.1"
+
+    input:
+    tuple val(meta), path(image), path(lesion) /* optional, input = [] */, path(fs_license) /* optional, input = [] */
+
+    output:
+    tuple val(meta), path("*__mask_wm.nii.gz")                , emit: wm_mask
+    tuple val(meta), path("*__mask_gm.nii.gz")                , emit: gm_mask
+    tuple val(meta), path("*__mask_csf.nii.gz")               , emit: csf_mask
+    tuple val(meta), path("*__parc.nii.gz")                   , emit: parc, optional: true
+    tuple val(meta), path("*__resampled_image.nii.gz")        , emit: resample, optional: true
+    tuple val(meta), path("*__vol.csv")                       , emit: vol, optional: true
+    tuple val(meta), path("*__qc.csv")                        , emit: qc, optional: true
```

Comment:
Expand the names to improve readability : `volume`, `qc_score`, `parcellation` or `gmparc`